.github/workflows/lint-and-test.yaml

```diff
@@ -2,6 +2,11 @@ name: Lint and test PR
 
 on: pull_request
 
+# Prevent to consume free CI minutes by not running two workflows in the CI at the same time.
+concurrency:
+  group: pr-testing
+  cancel-in-progress: true
+
```

Comment:
I think this may be problematic in some cases. e.g. if we're working in parallel or if renovate happens to open a PR while our tests are running. I don't see a huge benefit from this.

Reply:
And renovate opens all the PRs at the same time so they will block each other.

Good point. Removed in [#dc7fffa](https://github.com/txqueuelen/stateless-dns/pull/63/commits/dc7fffad3d2d50f6625e70e20764ec19b896da45)